roottest/python/JupyROOT/nbdiff.py

```diff
@@ -149,39 +133,53 @@ def addEtcToEnvironment(inNBDirName):
     return ipythondir
 
 
-def getInterpreterName():
-    """Find if the 'jupyter' executable is available on the platform. If
-    yes, return its name else return 'ipython'
-    """
-    ret = subprocess.call("type jupyter", shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    return "jupyter" if ret == 0 else "i%s" % pythonInterpName
-
-
 def getKernelName(inNBName):
     with open(inNBName) as f:
         nbj = json.load(f)
     if nbj["metadata"]["kernelspec"]["language"] == "python":
-        return pythonInterpName
+        return "python3"
```

Comment:
Is `python3` guaranteed to be available? i.e. is there any installation allowed to only install `python` rather than `pyhon3` (or both)?

Reply:
The `python3` kernel is always guaranteed to be available, because we only support Python 3 at this point. But you are right, this can be done better by updating the reference notebooks and taking the kernel name from the `kernelspec` directly.